spec/api/v1/schemas/hosts.rb

```diff
@@ -21,13 +21,13 @@ module Hosts
               example: true
             },
             os_major_version: {
-              type: 'string',
-              example: '7',
+              type: 'integer',
+              example: 7,
               nullable: true
             },
             os_minor_version: {
-              type: 'string',
-              example: '3',
+              type: 'integer',
+              example: 3,
```

Comment:
I decided to go ahead and update the type here to match inventory. I could revert it if you think it's a significant enough API breakage, but it seems pretty innocuous to me.